TestHelper.psm1

```diff
@@ -825,6 +825,69 @@ function Install-NugetExe
     Invoke-WebRequest -Uri $Uri -OutFile $OutFile
 } # Install-NugetExe
 
+<#
+    .SYNOPSIS
+        Gets the current Pester Describe block name
+#>
+function Get-PesterDescribeName
+{
+
+    return Get-CommandNameParameterValue -Command 'Describe'
+}
+
+<#
+    .SYNOPSIS
+        Gets the opt-in status of the current pester Describe 
+        block.  Writes a warning if the test is not opted-in.
+
+    .PARAMETER OptIns
+        An array of what is opted-in
+#>
+function Get-PesterDescribeOptInStatus
+{
+    param
+    (
+        [string[]]$OptIns
+    )
+
+    $describeName = Get-PesterDescribeName
+    $optin = $OptIns -icontains $describeName
```

Comment:
$optIn (upper 'I')  - Since you write the other as `$OptIns`
Same for all occurences